utils/src/userInput/showQuickPick.ts

```diff
@@ -62,7 +62,6 @@ export async function showQuickPick<TPick extends types.IAzureQuickPickItem<unkn
             // Show progress bar while loading quick picks
             quickPick.busy = true;
             quickPick.enabled = false;
-            quickPick.show();
```

Comment:
We were calling `show()` here so that while quick pick items are being created an empty quick pick with a progress bar is shown. We may need to call `show()` in both places.

Reply:
That solution doesn't seem to work unfortunately, I'll mess with it some more and see if there's another way we can handle this

Reply:
Seems like the issue is that if you call `show()` on a quick pick with no items, then items are added, focus is lost.

Reply:
I am also able to repro by just doing this:

![image](https://user-images.githubusercontent.com/40250218/204405818-3da4bd19-5d28-4616-9022-2822abd6de16.png)

Reply:
Nice 👍